DOCUMENTATION/content/guides/index.md

```diff
@@ -55,6 +55,10 @@ $root@server:~/laravel/ cd laradock
 $root@server:~/laravel/laradock# curl -L https://github.com/docker/compose/releases/download/1.8.0/run.sh > /usr/local/bin/docker-compose
 $root@server:~/chmod +x /usr/local/bin/docker-compose
 ```
+##  Enter the laradock folder and rename env-example to .env.
+```
+$root@server:~/laravel/laradock# cp env-example .env
```

Comment:
Could add suggestion to change information within the dotenv file in the future, however it's not necessary for now, I agree.